### ISSUE TITLE
Code Review

### ISSUE THREAD

**Issue by mutinhiri** [RUMBIDZAIDUMBU/Tigzozo-media#1]
I suggest that you begin implementing branching strategy when you are working you can have a dev branch where you will work and once a feature is done you can open a pull request and merge into the main branch

**Comment by RUMBIDZAIDUMBU** [comment]
I am not that familiar with GitHub Could you please schedule when you are free so that you can assist me because l have been uploading projects using gitonbrowser not good at using the terminal 

please assist
